src/views/TvsPageView.vue

```diff
@@ -0,0 +1,18 @@
+<template>
+  <div>
+    <tvs-list/>
+  </div>
+</template>
+
+<script>
+import TvsList from "@/components/Lists/TvsList";
```

Comment:
We don't need this view, **just import the component directly in the router, then delete the view**.